packages/plugin-css/src/index.ts

```diff
@@ -507,20 +515,25 @@ function getMode<T extends {id: string; $value: any; $extensions?: any; _origina
   return {value: token.$value, originalVal: token._original.$value};
 }
 
-/** convert token name to CSS variable */
-export function varName(id: string, options?: {prefix?: string; suffix?: string}): string {
-  return ['--', options?.prefix ? `${options.prefix.replace(DASH_PREFIX_RE, '').replace(DASH_SUFFIX_RE, '')}-` : '', id.replace(DOT_UNDER_GLOB_RE, '-'), options?.suffix ? `-${options.suffix.replace(DASH_PREFIX_RE, '')}` : ''].join('');
-}
-
 /** reference an existing CSS var */
-export function varRef(id: string, options?: {prefix?: string; suffix?: string; fallbacks?: string[]; mode?: string}): string {
+export function varRef(id: string, tokens: ParsedToken[], generateName: ReturnType<typeof makeNameGenerator>, suffix?: string): string {
```

Comment:
Also a note: will have to revert some of the changes to the `varRef` API as it’s used outside this plugin (may not be best practice, but it is what it is).

Mainly will revert the ordered params and revert back to a single object param for options (which is preferred in general, because then you can add/remove options easily, and handle backwards compatibility, with minimal effort)